hydra-node/src/Hydra/Network.hs

```diff
@@ -138,3 +138,8 @@ readPort s =
         else fail $ "readPort: " <> show n <> " not within " <> show maxPort
  where
   maxPort = fromIntegral (maxBound :: Word16)
+
+-- | Ports definition for Mock Chain client.
+-- HACK: This is a temporary solution until we wire in a real chain client.
+newtype MockChainPorts = MockChainPorts (PortNumber, PortNumber, PortNumber)
```

Comment:
The `HACK` note is true for the whole module, thus redundant. I'd be interested though why you opt for a `newtype` round triple instead of a 3-field record?

Reply:
ahh.. just saw the `auto` parser which uses `Read` :)

Reply:
given it's a stop-gap solution, I go for the "simplest thing that can work".